services/ui-src/src/components/fields/TextField.tsx

```diff
@@ -37,8 +38,13 @@ export const TextField = ({
     }
     // else set hydrationValue or defaultValue as display value
     else if (hydrationValue) {
-      setDisplayValue(hydrationValue);
-      form.setValue(name, hydrationValue, { shouldValidate: true });
+      if (props.clear) {
+        setDisplayValue(defaultValue);
+        form.setValue(name, defaultValue);
+      } else {
+        setDisplayValue(hydrationValue);
+        form.setValue(name, hydrationValue, { shouldValidate: true });
```

Comment:
Question: This is the only one of the other spots where we do this that we pass `shouldValidate: true`. Is there a reason for that?

Reply:
I'm not sure I follow your question, I see this on other field types.